src/components/RatingMenu.vue

```diff
@@ -52,11 +52,17 @@
 </template>
 
 <script>
-import algoliaComponent from '../component';
 import { connectStarRating } from 'instantsearch.js/es/connectors';
+import { createPanelConsumerMixin } from '../panel';
+import algoliaComponent from '../component';
 
 export default {
-  mixins: [algoliaComponent],
+  mixins: [
+    algoliaComponent,
+    createPanelConsumerMixin({
+      mapStateToCanRefine: state => !state.hasNoResults,
```

Comment:
I'm not sure about this one, does `hasNoResults` mean "no results with this refinement"? If that's the case it should be always `true`; since it's still possible to remove the refinement and give results again. 

It should be `false` if _none_ of the rating items can be applied I think